samples/highcharts/demo/combo-data-resolution/demo.js

```diff
@@ -0,0 +1,36 @@
+const data = JSON.parse(document.getElementById('data').textContent);
+
+Highcharts.chart('container', {
+    title: {
+        text: 'Global temperature mean anomalies from 1880 to 2016'
+    },
+    accessibility: {
+        description: `
+            A chart of monthly mean temperature anomalies from 1880 to 2016.
+            It employs two series, one column and one line,
+            to convey the same data at different resolutions.
+            There is only one column per thirty years,
+            but the line series have a datapoint per year.
+        `
+    },
+    series: [{
+        type: 'column',
+        data: data.map((d, i) => (i % 30 === 0) && d),
```

Comment:
I'd prefer if we used static data for the columns too. The purpose of this demo is to show how simple it is to set up Highcharts, so we should avoid too much fancy code. Especially since this is only about the demo data.